apps/docs/src/app/core/component-docs/carousel/examples/carousel-content-navigation-example.component.html

```diff
@@ -1,39 +1,39 @@
 <div style="display: flex; justify-content: center">
     <fd-carousel [navigatorInPageIndicator]="false" carouselIndicatorsOrientation="top" width="500px">
         <fd-carousel-item>
-            <img src="http://placeimg.com/500/500/nature/1" alt="first image" />
+            <img src="http://placeimg.com/500/500/nature/1" #imgInfo alt="first image" />
```

Comment:
I struggle to understand why this is needed.

Reply:
issue1
Below implementation is required in order for fd-carousel-item to have a title:
ngOnInit(): void {
if (!!this.imgInfoRef) {
this.title = this.imgInfoRef.nativeElement.getAttribute('alt');
}
}

Reply:
Please take a look at the my comment https://github.com/SAP/fundamental-ngx/pull/7083#discussion_r747422936

Reply:
Why does an img tag need a title? Isn't `alt` enough? Can you link a11y guidelines where it's said. you need additional title? Thank you!

Reply:
Why do we need two html elements, parent-child, having fist a title with some content and then alt with the same value?